src/draw/lv_draw_vector.c

```diff
@@ -206,9 +206,9 @@ void lv_matrix_transform_point(const lv_matrix_t * matrix, lv_fpoint_t * point)
 
 void lv_matrix_transform_path(const lv_matrix_t * matrix, lv_vector_path_t * path)
 {
-    for(uint32_t i = 0; i < path->points.size; i++) {
-        lv_fpoint_t * pt = lv_array_at(&path->points, i);
-        lv_matrix_transform_point(matrix, pt);
+    lv_fpoint_t * pt = lv_array_front(&path->points);
+    for(uint32_t i = 0; i < lv_array_size(&path->points); i++) {
```

Comment:
uint32_t size = lv_array_size(&path->points);
for(uint32_t i = 0; i < size; i++)